openedx_ledger/models.py

```diff
@@ -47,6 +49,7 @@ class Meta:
         editable=False,
         unique=True,
     )
+    history = HistoricalRecords()
```

Comment:
We don't need `history` on this abstract model, because it's a very general kind of mixin, simply providing modified /created timestamps and a uuid PK. The inheriting models should determine for themselves to add a history table.